packages/invoices/src/InboundShipment/DetailView/GeneralTab.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
```

Comment:
c+p..

Was unsure if inbound will be tabbed or not

Reply:
maybe not in phase one - there's the price tab; since that is fairly simple it may be ok to do in phase one